src/pages/NewChatPage.tsx

```diff
@@ -161,9 +166,12 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, translate, i
      * Creates a new 1:1 chat with the option and the current user,
      * or navigates to the existing chat if one with those participants already exists.
      *
-     * @param {Object} option
+     * @param option - Option choosen to create the chat
      */
-    const createChat = (option) => {
+    const createChat = (option: OptionData) => {
+        if (option.login === undefined || option.login === null) {
```

Comment:
I think in this case we can do it like that 
```suggestion
        if (!option.login) {
```
because if `option.login` is `''` we still dont want to navigate but I could be wrong here 😄